src/main/java/modbus/SlaveFolder.java

```diff
@@ -211,6 +221,70 @@ void makeEditAction() {
 		node.createChild(ACTION_EDIT, true).setAction(act).build().setSerializable(false);
 	}
 
+    private void makeExportAction(final Node fnode) {
+        Action act = new Action(Permission.READ, new Handler<ActionResult>(){
+            @Override
+            public void handle(ActionResult event) {
+                handleExport(fnode, event);
+            }
+        });
+        act.addResult(new Parameter("JSON", ValueType.STRING).setEditorType(EditorType.TEXT_AREA));
+        Node anode = fnode.getChild(ACTION_EXPORT, true);
+        if (anode == null) {
+            fnode.createChild(ACTION_EXPORT, true).setAction(act).build().setSerializable(false);
+        } else {
+            anode.setAction(act);
+        }
+    }
+
+    private void handleExport(Node fnode, ActionResult event) {
+        try {
+            Method serMethod = Serializer.class.getDeclaredMethod("serializeChildren", JsonObject.class, Node.class);
+            serMethod.setAccessible(true);
+            JsonObject childOut = new JsonObject();
+            serMethod.invoke(conn.link.serializer, childOut, fnode);
+            String retval = childOut.toString();
+            event.getTable().addRow(Row.make(new Value(retval)));
+        } catch (NoSuchMethodException | SecurityException | IllegalAccessException | IllegalArgumentException | InvocationTargetException e) {
+            LOGGER.debug("", e);
+        }
+    }
+
+    private void makeImportAction(final Node fnode) {
+        Action act = new Action(Permission.READ, new Handler<ActionResult>(){
+            @Override
+            public void handle(ActionResult event) {
+                handleImport(fnode, event);
+            }
+        });
+        act.addParameter(new Parameter("Name", ValueType.STRING));
+        act.addParameter(new Parameter("JSON", ValueType.STRING).setEditorType(EditorType.TEXT_AREA));
+        Node anode = fnode.getChild(ACTION_IMPORT, true);
+        if (anode == null) {
+            fnode.createChild(ACTION_IMPORT, true).setAction(act).build().setSerializable(false);
+        } else {
+            anode.setAction(act);
+        }
+    }
+
+    private void handleImport(Node fnode, ActionResult event) {
+        String name = event.getParameter("Name", ValueType.STRING).getString();
+        String jsonStr = event.getParameter("JSON", ValueType.STRING).getString();
+        JsonObject children = new JsonObject(jsonStr);
+        Node child = fnode.createChild(name, true).build();
+        try {
+            Method deserMethod = Deserializer.class.getDeclaredMethod("deserializeNode", Node.class, JsonObject.class);
+            deserMethod.setAccessible(true);
+            deserMethod.invoke(conn.link.deserializer, child, children);
+            SlaveFolder bd = new SlaveNode(conn, child);

```

Comment:
I believe this should be a `new SlaveFolder` - currently this creates a device node